uaa/src/test/java/org/cloudfoundry/identity/uaa/integration/OrchestratorZoneControllerIntegrationTests.java

```diff
@@ -141,9 +140,7 @@ public void testGetZone() {
 
             ConnectionDetails expectedConnectionDetails = new ConnectionDetails();
             expectedConnectionDetails.setSubdomain(zoneName);
-            expectedConnectionDetails.setUri("http://" + zoneName + ".localhost:8080/uaa");
             expectedConnectionDetails.setDashboardUri("http://localhost:8080/dashboard");
-            expectedConnectionDetails.setIssuerId(expectedConnectionDetails.getUri() + "/oauth/token");
```

Comment:
how we are validating uri and issuerId now?

Reply:
They are configured in uaa.yaml and integration_test_properties.yml. Right now both urls are configured with http://localhost:8080/uaa and it can't be changed now, since it affects other test cases. So I am just asserting both as same domain in the latest commit.